db/migrate/20180115012856_add_slug_to_admin_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddSlugToAdminUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :admin_users, :slug, :string, null: false
```

Comment:
__[RuboCop]__

Rails/NotNullColumn: Do not add a NOT NULL column without a default value.


<!-- Comment by [SideCI](https://sideci.com) -->